components/dada-execute/src/step.rs

```diff
@@ -386,6 +386,10 @@ impl<'me> Stepper<'me> {
                 permission: self.machine.new_permission(ValidPermissionData::our()),
             }),
             bir::ExprData::IntegerLiteral(v) => Ok(Value {
+                object: self.machine.new_object(ObjectData::Int(*v)),
+                permission: self.machine.new_permission(ValidPermissionData::our()),
+            }),
+            bir::ExprData::UnsignedIntegerLiteral(v) => Ok(Value {
```

Comment:
We have to modify `Stepper::apply_op` to determine what to do when (e.g.) two uints are added, or a uint plus an int, etc.

I am debating what the rules should be here!

I am thinking *maybe* we want to distinguish:

* signed integer (written `0i`)
* unsigned integer (written `0u`)
* unsuffixed integer (written `0`)

The idea would be that signed/unsigned can only mix with each other, but unsuffixed integers can mix with either.

We would then have some way to convert from signed to unsigned and vice versa.

My idea is this: I'm trying to move the rules that a static type system would impose to runtime.